src/Mumble.proto

```diff
@@ -10,14 +10,17 @@ package MumbleProto;
 option optimize_for = SPEED;
 
 message Version {
-	// 2-byte Major, 1-byte Minor and 1-byte Patch version number.
+	// Legacy version. 2-byte Major, 1-byte Minor and 1-byte Patch version number.
 	optional uint32 version = 1;
```

Comment:
```suggestion
	optional uint32 version_v1 = 1;
```
(The field's name are not part of the wire format and can thus be changed to our heart's content`
Also, thinking about it again, I think it might be better to group the different version fields together, even though the field ID would suggest a different ordering :thinking: